client/src/components/application/Stats/Stats.vue

```diff
@@ -131,7 +141,8 @@ export default {
     }
   },
   mounted() {
-    if (this.$refs.numberinput) this.$refs.numberinput.focus()
+    this.getPagedStats();
+    if (this.$refs.numberinput) this.$refs.numberinput.focus();
```

Comment:
Not a big deal but we don't use trailing commas. Funny the linter didn't flag these.

Reply:
Thanks for the fix.